.github/workflows/tests.yml

```diff
@@ -0,0 +1,139 @@
+name: API library tests
+
+on:
+  push:
+    branches:
+      - master
+      - '[0-9]+\.[0-9]+'
+  pull_request:
+  schedule:
+    # Run every day at 10 AM UTC to discover potential issues with recent changes to Mautic itself
+    - cron: '0 10 * * *'
```

Comment:
Hopefully this cron will allow us to discover potential issues with recent merges in `mautic/mautic` much earlier than we can now